docs/latest/examples/using-tailwindcss.md

```diff
@@ -0,0 +1,186 @@
+---
+description: |
+  Using Tailwind CSS in Fresh
+---
+
+Add Tailwind CSS to your project by importing the
+[Fresh Tailwind plugin](https://deno.land/x/fresh_tailwind) in your
+`fresh.config.ts` file.
+
+```ts fresh.config.ts
+import tailwindPlugin from "https://deno.land/x/fresh_tailwind/mod.ts";
+
+export default defineConfig({
+  plugins: [
+    tailwindPlugin(),
+  ],
+});
+```
+
+This is all that is required to add Tailwind styles. The plugin will compile
+Tailwind with every request if no plugin configuration is provided. This is
+useful for a quick development setup, but might not be necessary in a production
+environment.
+
+Provide the CSS source, or path to the source file, in the plugin's `css`
+option.
+
+```ts fresh.config.ts
+export default defineConfig({
+  plugins: [
+    tailwindPlugin({
+      css: "./src/style.css",
+    }),
+  ],
+});
+```
+
+The `dest` option can be specified to save output to a CSS file during
+[ahead-of-time builds](../concepts/ahead-of-time-builds.md).
```

Comment:
I can recommend running `deno task www` and clicking through things. This doesn't actually work! I think you need to remove the .md from the end.

Reply:
cool